cmd/main.go

```diff
@@ -135,6 +136,8 @@ func main() {
 
 	c.Flags().Uint64Var(&maxRetrySeconds, "max-retry-seconds", 7200, "Max retry seconds exponential backoff")
 
+	AddHCPVaultFlags(c)
```

Comment:
Is there a need to include them in a function like this?

Reply:
Well, this portion was getting too long so I thought I should just modularize this config

Reply:
Got it